lib/active_shipping/location.rb

```diff
@@ -73,10 +73,11 @@ def self.from(object, options = {})
 
       ATTRIBUTE_ALIASES.each do |attribute, aliases|
         aliases.detect do |sym|
-          value = if hash_access
```

Comment:
So if you have an active record object with `province` and `province_code`, because it has `hash_access`, we were preferring `province` instead of `province_code`.